trackpy/utils.py

```diff
@@ -269,3 +275,16 @@ def quiet(suppress=True):
         trackpy.logger.setLevel(logging.WARN)
     else:
         trackpy.logger.setLevel(logging.INFO)
+
+def _pandas_sort_pre_017(df, *args, **kwargs):
+    """Use sort() to sort a DataFrame"""
+    return df.sort(*args, **kwargs)
+
+def _pandas_sort_post_017(df, *args, **kwargs):
+    """Use sort_values() to sort a DataFrame"""
+    return df.sort_values(*args, **kwargs)
```

Comment:
Because the first argument changed its name from `columns` (in `df.sort`) to `by` (in `df.sort_values`), I think it would be better to make the call signature like this:

    pandas_sort(df, by, *args, **kwargs)

So that there cannot be any confusion.